FrontEnd/src/components/SignUp/components/signup/signup-form/SignUpFormContent.js

```diff
@@ -83,12 +84,18 @@ export function SignUpFormContentComponent(props) {
     axios({
       method: 'post',
       url: `${process.env.REACT_APP_BASE_API_URL}/api/auth/users/`,
-      withCredentials: false,
-      data: dataToSend
-    }).then(res => console.log(res.data)).catch(error => console.log(error));
-    console.log(process.env.REACT_APP_BASE_API_URL);
+      data: dataToSend,
+    })
+      .then()
```

Comment:
No need to leave empty `then` here

Reply:
But should the user somehow be informed that registration is successfull?

Reply:
It will be used to trigger notification modal #262 as far as I know. That's why I decided to leave it even though it's empty.